lib/dart_libraries_for_browser_types.ts

```diff
@@ -1,9 +1,7 @@
-import ts = require('typescript');
-
 // Autogenerated list. Do not edit by hand.
 // Open DartBrowserMapping.dart from the Dart SDK in Dartium to regenerate this
```

Comment:
you could remove this comment now as it is bogus as dartium doesn't run anymore. you might be able to find the DartBrowserMapping file. Terry probably also has some better ideas on how to implement it.